code/game/objects/items/weapons/twohanded.dm

```diff
@@ -179,6 +179,7 @@
 	slot_flags = SLOT_BACK
 	force_unwielded = 5
 	force_wielded = 24
+	toolspeed = 0.25
```

Comment:
Isn't this a general buff to fireaxes?

Reply:
No, toolspeed isnt used by fireaxes anywhere before this pr.